docs/DeveloperGuide.adoc

```diff
@@ -523,18 +523,18 @@ These operations are exposed in the `TabManager` class respectively as:
 
 The following Activity Diagrams illustrate what happens when the user executes a `SwitchTabCommand` or `OpenTabCommand`
 
-image::gui/SwitchTabCommandActivityDiagram.png[]
+image::gui/SwitchTabCommandActivityDiagram.png[width="50%"]
```

Comment:
Just FYI, setting the width fixed the display issue on Web but the PDF export would probably be ruined...